tests/test_waveplate_thorlabs_kb10crm.py

```diff
@@ -0,0 +1,9 @@
+from pnpq.devices.waveplate_thorlabs_kb10crm import Waveplate
+from pnpq.errors import DevicePortNotFoundError
+
+import pytest
+
+
+def test_disconnected_initialization():
+    with pytest.raises(DevicePortNotFoundError):
+        Waveplate("ABC", "DEF")
```

Comment:
For Thorlabs K10CR1 waveplate rotator, the device serial number starts with '55' and is in this format: 55xxxxxx

Reply:
That's good to know for the future.